README.md

```diff
@@ -1,18 +1,18 @@
 # SwarmDB
 
-(name change in process; for future reference please use: https://swarmdb.dev/ which we will keep up to date even if the repo name changes)
+(name change in process; for future reference please use: https://swarmdb.dev which we will keep up to date even if the repo name changes)
 
 ## What is SwarmDB?
 
-An encrypted, eventually consistent object real-time application database with dynamic access control using the distributed web as a storage layer. Then intent is to make it easy to start with a secure, private, local database, and scale as needed in a performant manner without the requirement of adding or moving to a centrally managed database.
+An open-source, encrypted, eventually consistent object real-time application database with dynamic access control using the distributed web as a storage layer. Then intent is to make it easy to start with a secure, private, local database, and scale as needed in a performant manner without the requirement of adding or moving to a centrally managed database.
```

Comment:
Also, thinking `real-time` maybe doesn't explain the asynchronous nature, wondering if you @robotoer  have thoughts/suggestions on that part of the wording

Reply:
Maybe reference the ability to build collaborative/offline first applications. Those are terms developers should understand